docs/userguide.md

```diff
@@ -117,9 +135,9 @@ Next, you'll want to use the `kubefedctl` tool to join all your
 clusters that you want to test against.
 
 ```bash
-./bin/kubefedctl join cluster1 --cluster-context cluster1 \
```

Comment:
Why removing `./bin` here?

Reply:
It is useless when we set bin as part of `PATH` in prerequisite.
Remove it from doc to make `kubefedctl` usage consistent across document.